pkg/project/auth/cache_test.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
```

Comment:
i need to add more tests that use policy objects directly instead of mock reviewer.

i want integration tests too.

card will be added.

i need more time in a day ;-)